orders/templates/orders/emails/order-confirmation.txt

```diff
@@ -1,6 +1,10 @@
 Hi {{ purchaser_name }},
 
+{% if num_tickets %}
 You have purchased {{ num_tickets }} ticket{{ num_tickets|pluralize }} for PyCon UK 2018.
+{% else %}
+You have purchased {{ num_items }} item{{ num_items|pluralize }} for PyCon UK 2018.
+{% endif %}
```

Comment:
It'd be good to list what the items are, if possible.

Reply:
I'll have a look - we put it in the receipt, so I can't see why we can't put it here.